GSOC18/main.py

```diff
@@ -0,0 +1,80 @@
+from hexCells_to_squareCell_interpolation import linear_interpolate_hex_to_square
+import sys
+sys.path.append('/home/abhinav/Desktop/HAhRD/GSOC18/hex_cells_data/cells_out.pkl')
```

Comment:
Please ignore this line. I don't know why it is not able to add the path. Also, the directory structure is changed from here.